support/doc/tools.md

```diff
@@ -303,6 +303,33 @@ $ cd /var/www/peertube-docker
 $ docker-compose exec -u peertube peertube npm run create-import-video-file-job -- -v [videoUUID] -i [videoFile]
 ```
 
+### create-move-video-storage-job.js
+
+Use this script to move a specific video file in object storage.
```

Comment:
Use this script to move all video files, or a specific video file, to object storage.